site/content/how-to/monitoring/tracing.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "Tracing"
+description: "Learn how to configure tracing in NGINX Gateway Fabric."
```

Comment:
```suggestion
```

We shouldn't be using the description field, as although it currently displays at the top of pages, this is actually a bug and will create extra work in the future once it's fixed. I've added a suggestion to add this to the top of the page in the body text.